arch/arm64/boot/dts/qcom/sdm660-xiaomi-clover-plus.dts

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, Mark Hargreaves <clashclanacc2602@gmail.com>
+ */
+
+/dts-v1/;
+#include "sdm660-xiaomi-clover-common.dtsi"
+
+/ {
+	model = "Xiaomi Mi Pad 4 Plus";	
+	compatible = "xiaomi,clover-plus", "qcom,sdm660", "qcom,sdm660-mtp", "qcom,mtp";
+	vreg_bl_vddio: lcd-backlight-vddio {
+		compatible = "regulator-fixed";
+		regulator-name = "vreg_bl_vddio";
+		regulator-min-microvolt = <3150000>;
+		regulator-max-microvolt = <3150000>;
+
+		gpio = <&tlmm 72 0>;
+		enable-active-high;
+
+		startup-delay-us = <70000>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&lcd_bl_en_default>;
+	};
+};
+
+&blsp_i2c3 {
+	status = "okay";
+	touchscreen@5d {
+		compatible = "goodix,gt9110";
+		reg = <0x5d>;
+		AVDD28-supply = <&vreg_l3b_3p3>;
+		VDDIO-supply = <&vreg_l11a_1p8>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <67 IRQ_TYPE_LEVEL_LOW>;
+		pinctrl-names = "default", "int-output-low", "int-output-high", "int-input";
+		pinctrl-0 = <&ts_int_default>;
+		pinctrl-1 = <&ts_int_output_low>;
+		pinctrl-2 = <&ts_int_output_high>;
+		pinctrl-3 = <&ts_int_input>;
```

Comment:
weird, 4 pinctrls? Usually there are 2, "default" when device is on, and "sleep" for suspended/off state